spec.bs

```diff
@@ -1014,15 +1014,14 @@ special value "*". The only interfaces that can be exposed inside of the globals
 specification are those that explicitly list the global names provided here.
 
 <pre class="idl">
-[Global=InterestGroupScriptRunnerGlobalScope,
- Exposed=InterestGroupScriptRunnerGlobalScope]
+[Exposed=InterestGroupScriptRunnerGlobalScope,
+ Global=InterestGroupScriptRunnerGlobalScope]

```

Comment:
I think @domfarolino was suggesting removing this Global= attribute, but keeping it on the three derived interfaces.

Reply:
Thanks. Updated. I don't know how I read it wrong :(